integration-libs/opf/base/root/services/opf-global-message.service.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  StateWithGlobalMessage,
+  Translatable,
+} from '@spartacus/core';
+import { timer } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OpfGlobalMessageService extends GlobalMessageService {
```

Comment:
[MINOR]
This service is a workaround for an issue that is not obvious. Let's explain it in the JSDocs what would happen in OPF, if we didn't have this service.